lighthouse-core/audits/service-worker.js

```diff
@@ -29,17 +29,30 @@ class ServiceWorker extends Audit {
    * @return {LH.Audit.Product}
    */
   static audit(artifacts) {
-    // Find active service worker for this URL. Match against
+    const {versions, registrations} = artifacts.ServiceWorker;
+    const pageUrl = new URL(artifacts.URL.finalUrl);
+
+    // Find active service workers for this origin. Match against
     // artifacts.URL.finalUrl so audit accounts for any redirects.
-    const versions = artifacts.ServiceWorker.versions;
-    const url = artifacts.URL.finalUrl;
+    const matchingSWVersions = versions.filter(v => v.status === 'activated')
```

Comment:
do we want to consider 'activating' or even 'installed'/'installing' ?
I can't really tell if including those cases could give a false positive.

Reply:
I'm not sure. The gatherer runs in `beforePass` of the offline pass...if it's just installing I don't think it'll be ready to serve assets for that pass. I'm not even sure if this is a possible state...if you're installing and a navigation to `about:blank` and back occurs, does it keep installing?

Reply:
k sg. since we're in beforePass of offline, this seems entirely reasonable.